### PR TITLE
Include hashed user id in session tokens

### PR DESCRIPTION
**Pull request by swemail**
WAIT FOR #1535 and bump of killswitch!!!!

Include a hash based in room id and user id (+ DAILY_ID_SALT) in the session tokens. When user id is not included in tokens the `user_id` and `session_id` are the same on a daily participant. When user id is included in the tokens the `user_id` and `session_id` are not the same. `active-speaker-change` sends what it calls `peerId` witch is the same as `session_id` so the participant sort order have to find the user based on the `session_id` and therefor setting the order is moved into `dailyState` where the current state can be accessed. Since `session_id` is ephemeral using that for sort order is not stable.

### CODE REVIEW

File: functions/src/lib/id.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+import config from './config';
+
+const {DAILY_ID_SALT} = config;
+
+const uuidv5Bytes = (name: string) => uuidv5(name, DAILY_ID_SALT, []);
+const createId = compose(bs58.encode, uuidv5Bytes);
+
+export const createRctUserId = (roomId: string, userId: string) =>
+  createId(`room:${roomId}:user:${userId}`);
```
Comment (gewfy) [inline functions/src/lib/id.ts:13]: Nit pick, but what is `rct`? Wouldn't `daily` make more sense?

**Review by gewfy — APPROVED** [review]
Nice nice nice nice!

File: functions/src/lib/id.ts (hunk 1 above)
Comment (swemail) [reply functions/src/lib/id.ts:13]: Haha, yes, reused the mighty Örjans naming convention. Make kind of sense, they migrated video platform and maybe we will too 😄

Comment (swemail) [reply functions/src/lib/id.ts:13]: rct, `real time communication` apparently

**Pull request merged**